DASH-IF-Ingest.bs.md

```diff
@@ -545,38 +545,28 @@ DASH-IF makes no warranty whatsoever for such third party material.
 
 # General Ingest Protocol Behavior # {#general}
 
-  The media ingest follows the following  
-  general requirements for both targetted interfaces.  
-  The ingest interface uses existing media formats 
-  in combination with the HTTP POST method.
+  Live media ingest follows the following  
+  general requirements. It is recommended that 
+  a conforming implementation fullfills these requirements.
 
-     1. The [=Ingest source=]  SHALL communicate
-        using the HTTP POST method as defined in
+     1. The [=Ingest source=] SHALL communicate
+        using the HTTP POST or PUT method as defined in
         the HTTP protocol, version 1.1 [[!RFC7235]]
-     2. The [=Ingest source=]  SHOULD
+     2. The [=Ingest source=] SHOULD
         use HTTP over TLS, if TLS is used it SHALL support atleast
-        TLS version 1.2, higher version
-        may also be supported additionally [[!RFC2818]]
-     3. The [=Ingest source=]  SHOULD us a domain name system for
-        reolving hostnames to ip addresses such as
-        DNS [[!RFC1035]] or any other system that is in place.
-        If this is not the case, the domain names the ip adress mapping 
-        must be known and static, such as configured in the operating system.
+        TLS version 1.2 [[!RFC2818]]
```

Comment:
This reference should be updated as RFC 2818 is an informational not standards track document defining the use of HTTP with TLS, if [RFC 5246](https://tools.ietf.org/html/rfc5246) is not appropriate, consider referencing [Section 2.7.2 of RFC 7230](https://tools.ietf.org/html/rfc7230#section-2.7.2)